enterprise/admin/views.py

```diff
@@ -791,7 +792,15 @@ def _enroll_users(
                 pending_messages.append(cls.get_failed_enrollment_message(failed, program_identifier))
             if pending:
                 pending_messages.append(cls.get_pending_enrollment_message(pending, program_identifier))
-
+        
+        enrollments = map(lambda success: {
+            "lms_user_id": success.id,
+            "email": success.email,
+            "username": success.username,
+            "course_run_key": course_id,
+        }, succeeded)
```

Comment:
While `map`s work and make sense from a functional point of view, a list comprehension here is a little more in line with the rest of our code base and they're generally more "pythonic". 

Fun fact: `map`, `reduce` and `filter` were all actually going to be removed from Python 3, but some people got real mad about that, so they stayed.

Reply:
Interesting - I was going to ask this question earlier today, but decided "I'll just write it like a JS developer and someone will correct me."

I'll refactor it - good learning experience 😃

Reply:
Refactored!